src/open_inwoner/scss/components/Header/PrimaryNavigation.scss

```diff
@@ -233,20 +240,19 @@
 
   /// Interaction.
 
-  &:not(#{&}--dismissed) &__list-item:focus &__list,
-  &:not(#{&}--dismissed) &__list-item:focus-within &__list,
+  &:not(#{&}--dismissed) &__list-item &__list:focus-within,
   &:not(#{&}--dismissed)[class*='--open'] &__list-item &__list {
     transform: scaleY(1);
   }
-  &:not(#{&}--dismissed) &__list-item:focus,
-  &:not(#{&}--dismissed) &__list-item:focus-within {
+
+  &:not(#{&}--dismissed)[class*='--open'] &__list-item {
```

Comment:
Just a note, but selectors like `&:not(#{&}--dismissed)[class*='--open']` have that little whiff that triggers the senses.